packages/server/src/lib/targeting.test.ts

```diff
@@ -34,36 +34,12 @@ describe('audienceMatches', () => {
         expect(got).toBe(true);
     });
 
-    it('returns false for recent contributor if test is for AllExistingSupporters', () => {
-        const got = audienceMatches(
-            false,
-            'AllExistingSupporters',
-            '2022-01-01',
-            new Date('2022-01-02'),
-        );
-        expect(got).toBe(false);
-    });
-    it('returns false for recent contributor if test is for AllNonSupporters', () => {
-        const got = audienceMatches(
-            false,
-            'AllNonSupporters',
-            '2022-01-01',
-            new Date('2022-01-02'),
-        );
-        expect(got).toBe(false);
-    });
-
     it('returns true for old contributor if test is for AllNonSupporters', () => {
```

Comment:
I think you can also delete these 2 `old contributor` tests, the `audienceMatches` function no longer knows if they're a single contributor